### PR TITLE
Remove LIB environment variable [DOC-2449]

### PR DESCRIPTION
**Pull request by SmritiSatya**
Remove LIB environment variable from chaos faults
JIRA ticket: https://harness.atlassian.net/browse/DOC-2449
Signed-off-by: Smriti S <[smriti.satyanarayana@harness.io]>

### CODE REVIEW

File: docs/chaos-engineering/chaos-faults/categories.tsx [hunk 1]
```diff
@@ -14,12 +14,12 @@ export const categories: FaultCardItem[] = [
...
-    description: <>Short description about this</>,
-    faults: 36,
-    category: "linux",
-  },
+//  {
+//    title: "Linux",
+//    description: <>Short description about this</>,
+//    faults: 36,
+//    category: "linux",
+//  },
```
Comment (neelanjan00) [inline docs/chaos-engineering/chaos-faults/categories.tsx:22]: I think Sahil has added this, we can ask him to remove it if we don't need it.

File: docs/chaos-engineering/chaos-faults/kubernetes/pod/static/manifests/container-kill/pumba.yaml [hunk 2]
```diff
@@ -17,8 +17,6 @@ spec:
       components:
         env:
         # name of the lib
-        # supoorts pumba and litmus
-        - name: LIB
-          value: 'pumba'
+        # supports pumba and litmus
```
Comment (neelanjan00) [inline docs/chaos-engineering/chaos-faults/kubernetes/pod/static/manifests/container-kill/pumba.yaml:20]: We can remove this file as we don't need it

File: docs/chaos-engineering/chaos-faults/static/manifest/common/lib.yaml [hunk 3]
```diff
@@ -12,10 +12,4 @@ spec:
     appkind: "deployment"
   chaosServiceAccount: litmus-admin
   experiments:
-  - name: pod-delete
```
Comment (neelanjan00) [inline docs/chaos-engineering/chaos-faults/static/manifest/common/lib.yaml:15]: We can remove this file entirely, since it has no purpose now.

**Comment by SahilKr24** [comment]
image

**Pull request merged**